db/migrate/20240412064149_add_jti_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddJtiToUsers < ActiveRecord::Migration[7.0]
```

Comment:
It can't be helped in this case, but in general we should ensure that migrations can be run directly without having to drop the local database, since that would mean that it would not be able to be run on a production environment. In this case, adding `default: ""` to the migration would allow it to be executed for existing seeded users.